packages/react/src/helpers/utils.ts

```diff
@@ -1 +1,7 @@
 export const isDevelopment = () => process.env.NODE_ENV !== 'production';
+
+export const isInputEventTarget = (
+  target: unknown
+): target is HTMLInputElement => {
+  return (target as HTMLInputElement)?.nodeName === 'INPUT';
+};
```

Comment:
Was there a bug or type issue?  I'm curious what you saw that we may have missed so far...

Reply:
Yes, changing from `Form` to `form` exposed that the `handleChange` function had the wrong type on its `event` parameter, which caused type errors on `form.onChange` and when trying to de-structure the `event.target`. The form onchange event doesn't know what type of element fired the change event, so the `isInputEventTarget` typeguard ensures it was an input element.

```diff
-  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+ const handleChange = (event: React.FormEvent<HTMLFormElement>) => {
```